train.py

```diff
@@ -284,12 +285,14 @@ def loss_fn(pred, labels):
                     # need to free to before bwd to avoid peaking memory
                     del pred
                     loss.backward()
+                
+                for m in model_parts:
```

Comment:
No need for changes, but for documentation purposes, when did we observe overflow here?

Reply:
This is a problem, especially at the beginning of the training when the grades are big. Don't recall the exact instance but as soon as you remove it everything is smooth.